README.md

```diff
@@ -4,33 +4,33 @@
+    * [1.5. Tips on Tagging](./build/Build.md#tips-on-tagging)
+    * [1.6. Vulnerability scanning and container security check](./build/Build.md#scanning)
+    * [1.7. Image signing and Verification](./build/Build.md#signing)
 
-- ### [2. Pushing your image to a container registry](https://github.com/wso2/container-guide)
-    * [2.1. Why should you have your own private registry ?](https://github.com/wso2/container-guide)
-    * [2.2. How to secure your private registry ?](https://github.com/wso2/container-guide)
-    * [2.3. Scanning and Auditing](https://github.com/wso2/container-guide) 
+- ### [2. Pushing your image to a container registry](./push/Push.md)
+    * [2.1. Why should you have your own private registry ?](./push/Push.md#why-private-registry)
+    * [2.2. How to secure your private registry ?](./push/Push.md#how-to-secure)
+    * [2.3. Scanning and Auditing](./push/Push.md#scanning-auditing) 
 
-- ### [3. Deploying into production](https://github.com/wso2/container-guide)
-    * [3.1. Why do you need a container orchestrator ?](https://github.com/wso2/container-guide)
-    * [3.2. Why should you choose Kubernetes ?](https://github.com/wso2/container-guide)
-    * [3.3. Kubernetes Vs. OpenShift ?](https://github.com/wso2/container-guide)
-    * [3.4. Helm as package manager](https://github.com/wso2/container-guide) 
-    * [3.5. Supporting Infrastructure](https://github.com/wso2/container-guide)
-    * [3.6. Resource Allocation](https://github.com/wso2/container-guide)
-    * [3.7. Deployment definition tips](https://github.com/wso2/container-guide) 
-    * [3.8. Managing configurations](https://github.com/wso2/container-guide)  
-    * [3.9. Managing keystores, certificates and other artifacts](https://github.com/wso2/container-guide/blob/master/deploy/Managing_Keystores_And_Truststores.md)
-    * [3.10. Managing databases](https://github.com/wso2/container-guide)
-    * [3.11. Production deployment tips](https://github.com/wso2/container-guide)
-    * [3.12. Managing security](https://github.com/wso2/container-guide)
+- ### [3. Deploying into production](./deploy/Deploy.md)
+    * [3.1. Why do you need a container orchestrator ?](./deploy/Deploy.md)
+    * [3.2. Why should you choose Kubernetes ?](./deploy/Deploy.md)
+    * [3.3. Kubernetes Vs. OpenShift ?](./deploy/Deploy.md)
+    * [3.4. Helm as package manager](./deploy/Deploy.md) 
+    * [3.5. Supporting Infrastructure](./deploy/Deploy.md)
+    * [3.6. Resource Allocation](./deploy/Deploy.md)
+    * [3.7. Deployment definition tips](./deploy/Deploy.md) 
+    * [3.8. Managing configurations](./deploy/Deploy.md)  
+    * [3.9. Managing keystores, certificates and other artifacts](./deploy/Deploy.md)
+    * [3.10. Managing databases](./deploy/Deploy.md)
+    * [3.11. Production deployment tips](./deploy/Deploy.md)
+    * [3.12. Managing security](./deploy/Deploy.md)
 
 - ### [4. Storage](https://github.com/wso2/container-guide/blob/master/store)
```

Comment:
@DilanUA shall we please fix the other links as well to make then consistent with chapters 1, 2 and 3?

Reply:
@chirangaalwis Done